node/bft/src/gateway.rs

```diff
@@ -131,6 +131,9 @@ pub struct Gateway<N: Network> {
     /// prevent simultaneous "two-way" connections between two peers (i.e. both nodes simultaneously
     /// attempt to connect to each other). This set is used to prevent this from happening.
     connecting_peers: Arc<Mutex<IndexSet<SocketAddr>>>,
+    #[cfg(feature = "telemetry")]
+    /// The validator telemetry.
```

Comment:
```suggestion
    /// The validator telemetry.
    #[cfg(feature = "telemetry")]
```
the established Rust convention

Reply:
Resolved with [2ba93d8](https://github.com/ProvableHQ/snarkOS/pull/3516/commits/2ba93d833ab3a0017bca6a071d8260838eac4521)